butterfly-cli/src/test/resources/helpOut.txt

```diff
@@ -20,7 +20,7 @@ Option                           Description
                                    application is placed under a new folder    
                                    whose name is same as original folder, plus 
                                    "-transformed-yyyyMMddHHmmssSSS" suffix     
--p <inline properties>           Transformation specific properties, used to   
+-p <String: inline properties>   Transformation specific properties, used to   
```

Comment:
What is the reason for this change? This is CLI help information. How adding `String` here would help an user when typing the command in the CLI? By the way, String is a Java specific term but the CLI user doesn't even have to know that.

Reply:
newer version of jopt insists on showing the type. there does not seem to be an easy way to disable that.

see:
[new version](https://github.com/jopt-simple/jopt-simple/blob/jopt-simple-5.0.4/src/main/java/joptsimple/BuiltinHelpFormatter.java#L436-L443)
[current version](https://github.com/jopt-simple/jopt-simple/blob/jopt-simple-5.0.2/src/main/java/joptsimple/BuiltinHelpFormatter.java#L432-L439)

Reply:
Ok, thanks